app/com/arpnetworking/kairos/client/models/Metric.java

```diff
@@ -161,6 +161,23 @@ public Builder setOrder(@Nullable final Order value) {
             return this;
         }
 
+        /**
+         * Static factory initializes a builder with state from
+         * the source instance.
+         *
+         * @param source Metric to copy values from
+         * @return Builder resulting builder with values copied
+         */
+        public static Builder fromMetric(final Metric source) {
```

Comment:
There should already be a clone method that will do this for you.

Reply:
Are you talking about the ThreadLocalBuilder one?

Reply:
This one: https://github.com/ArpNetworking/commons/blob/master/src/main/java/com/arpnetworking/commons/builder/OvalBuilder.java#L74

Reply:
Fixed